spec/file-spec.coffee

```diff
@@ -28,6 +28,15 @@ describe 'File', ->
   it 'returns false from isDirectory()', ->
     expect(file.isDirectory()).toBe false
 
+  describe '::isSymbolicLink', ->
+
```

Comment:
My kingdom for an autofix linter. Machines should be doing these things for us! But yup, I'll remove.